Source/Core/TestCase.jsl

```diff
@@ -750,15 +750,13 @@ ut test assertion decorator = Function( {function name, args},
 			Insert Into( inner call, Eval Expr( Name Expr( Expr( Name Expr( arg ) ) ) ) )
 		);
 	);
-	Eval Expr(
-		As Scoped( local, Expr( function name ) ) = Function( Expr( args ),
+	Parse( Eval Insert( "
+		local:^function name^ = Function( ^Char(args)^,
```

Comment:
`Char()` is inferred within `EvalInsert`, so it can be removed if desired to make it cleaner. Also valid for the Char on line 756.